pkg/controller/registry/grpc/source.go

```diff
@@ -147,7 +147,8 @@ func getEnvAny(names ...string) string {
 }
 
 func grpcConnection(address string) (*grpc.ClientConn, error) {
-	dialOptions := []grpc.DialOption{grpc.WithInsecure()}
+	creds := insecure.NewCredentials()
```

Comment:
Are these changes required because of what the lint took found?

Reply:
yup

Reply:
👍